packages/slice-machine/lib/builders/SliceBuilder/SideBar/components/ImagePreview.tsx

```diff
@@ -16,11 +16,11 @@ interface ImagePreviewProps {
   src?: string;
   onScreenshot: () => void;
   imageLoading: boolean;
-  onHandleFile: (file: File) => void;
```

Comment:
it's because atm there is still some uncertainty on how to handle the "custom screenshot" button. So atm in the changes page we are not rendering it. The button is conditionally rendered if the `onHandleFile` method is defined.
<img width="392" alt="image" src="https://user-images.githubusercontent.com/69144671/184892144-55c54bc8-422b-474f-99a7-640fc5b058b7.png">